Cargo.toml

```diff
@@ -115,7 +115,7 @@ winapi = "0.3"
 hassle-rs = "0.10.0"
 
 # Gles dependencies
-khronos-egl = "4.1"
+khronos-egl = ">=4.1,<7"
```

Comment:
This can now be just `7`. Firefox has committed to windows-sys (but not windows) 0.48.

Reply:
v6 you mean.
Done!